webapp/app/forms/validators.py

```diff
@@ -115,3 +117,30 @@ def __call__(self, form, field):
         for char in input_str:
             if char not in VALID_UNLOCK_CODE_CHARACTERS:
                 raise ValidationError(_('validate.invalid-character'))
+
+class ValidDayOfBirth:        
+    def __call__(self, form, field):
+        raw_input_date = field._value()
+
+        if isinstance(raw_input_date, str):
+            day, month, year = raw_input_date.split('.')
+        elif isinstance(raw_input_date, list):
+            day, month, year = raw_input_date
+        
+        elster_min_date = datetime.date(1900, 1, 1)
+
+        if not day or not month or not year: 
+            raise ValidationError(_('validation-dob-incomplete'))
+            
+        try:
+            input_date = datetime.date(int(year), int(month), int(day))
+
+            if input_date > datetime.date.today():
+                raise ValidationError(_('validation-dob-in-the-future'))
+
+            if input_date < elster_min_date:
+                raise ValidationError(_('validation-dob-to-far-in-past'))
+        except ValidationError as e:
+            raise e
+        except ValueError:
+            raise ValidationError(_('validation-dob-incorrect'))
```

Comment:
Can we just wrap the one relevant line: l. 136 in the try-except-block? the `except ValidationError` part confuses me^^